### ISSUE TITLE
duplicate

### ISSUE THREAD

**Issue by vkshaw0807** [Feodor2/Mypal68#513]
MyPal 68.14.4b get slower after using 1 GB of Memory. 
After using some time it shows glitches which is stop only after restarting the browser.
how to solve???? 
![MyPal 68 14 4b memory use](https://github.com/user-attachments/assets/fc601fc8-2fb4-4e65-839a-8c4275f0b280)

**Comment by bluedragon-cairo** [comment]
This happens in all Firefox based 32bit browsers, also in New Moon and Mypal 28/29.

**Comment by bluedragon-cairo** [comment]
maybe duplicate of #466?
